<!-- <?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginButton_2</class>
 <widget class="QDialog" name="loginButton_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                          stop:0 #2c3e50, stop:1 #34495e);
        font-family: "Segoe UI", sans-serif;
        font-size: 14px;
        color: #ecf0f1;
    }

    QLabel {
        color: #ecf0f1;
        font-weight: bold;
    }

    QLineEdit {
        background-color: #1e272e;
        border: 1px solid #7f8c8d;
        border-radius: 5px;
        padding: 5px;
        color: #ecf0f1;
    }

    QPushButton {
        background-color: #16a085;
        color: white;
        padding: 6px 12px;
        border-radius: 5px;
        font-weight: bold;
    }

    QPushButton:hover {
        background-color: #1abc9c;
    }

    QPushButton:pressed {
        background-color: #149174;
    }

    QPlainTextEdit {
        background-color: transparent;
        border: none;
        color: #bdc3c7;
        font-style: italic;
    }
   </string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Signup Here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgotPasswordButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Forgot Password?</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="plainText">
    <string>No account?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui> -->
